packages/react-core/src/demos/ComposableMenu/examples/ComposableTreeViewMenu.tsx

```diff
@@ -168,10 +169,19 @@ export const ComposableTreeViewMenu: React.FunctionComponent = () => {
       return;
     }
     if (menuRef.current.contains(event.target as Node) || toggleRef.current.contains(event.target as Node)) {
-      if (event.key === 'Escape' || event.key === 'Tab') {
+      if (event.key === 'Escape') {
```

Comment:
I think a comment before the `handleMenuKeys` function to explain the goals or use cases covered in this function would be super helpful.